### PR TITLE
chore(ci): fast fail travis script in case of error

### PR DESCRIPTION
**Pull request by sonasingh46**
Signed-off-by: Ashutosh Kumar <ashutosh.kumar@mayadata.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ before_script: # TODO add golangci yaml config
 script:
   - make test
   - if [ "$TRAVIS_CPU_ARCH" == "amd64" ]; then
-      make all.amd64;
+      make all.amd64 || exit 1;
```
Comment (prateekpandey14) [inline .travis.yml:54]: we can use `travis_terminate 1;`

Comment (sonasingh46) [reply .travis.yml:54]: No documentation found regarding that. 
Found the `travis_terminate 1` thing in a deprecated api.
I asked the same in the Travis community.
https://travis-ci.community/t/how-to-stop-the-execution-of-travis-if-script-exits-with-an-error/8556

File: ci/sanity/install.sh [hunk 2]
```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+
```
Comment (kmova) [inline ci/sanity/install.sh:2]: though not related to current PR, whenever we are touching any existing files, if a copyright message is missing, let us add one.

File: .travis.yml [hunk 3]
```diff
@@ -1,3 +1,17 @@
+# Copyright © 2020 The OpenEBS Authors
```
Comment (kmova) [inline .travis.yml:1]: :+1:

**Pull request merged**